### PR TITLE
docs(@clayui/table): Update Cell's Dropdown component

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

File: packages/clay-table/stories/index.tsx [hunk 1]
```diff
@@ -31,41 +31,23 @@ function ClayCheckboxWithState(props: any) {
...
-		</DropDownWithState>
+		<ClayDropDownWithItems
+			items={[
+				{href: '#1', label: 'One'},
+				{href: '#2', label: 'Two'},
+				{href: '#3', label: 'Three'},
+			]}
+			spritemap={spritemap}
+			trigger={
+				<ClayButton className="component-action" monospaced>
```
Comment (matuzalemsteles) [inline packages/clay-table/stories/index.tsx]: Add `displayType="unstyled"`, the button flashes with the blue background in the hover.

Comment (diegonvs) [reply packages/clay-table/stories/index.tsx]: How did you catch it? Ninja

Comment (matuzalemsteles) [reply packages/clay-table/stories/index.tsx]: Actually not the hover is when you click the button. 🙂

**Comment by matuzalemsteles** [comment]
hey @diegonvs looks like the blue flash on the DropDown button is still going on.

**Pull request merged**